libzseek.pc.in

```diff
@@ -7,7 +7,7 @@ Name: libzseek
 Description: Random access decompression library
 Version: @VERSION@
 Requires:
-Requires.private: libzstd >= 1.4.9
+Requires.private: libzstd >= 1.4.9 liblz4 >= 1.7.5
```

Comment:
Let's bump this to 1.8.3 as well

Reply:
Missed that, thank you for pointing it out!